engine/src/main/java/org/hibernate/search/v6poc/engine/impl/SearchMappingRepositoryBuilderImpl.java

```diff
@@ -93,7 +93,7 @@ public SearchMappingRepository build() {
 		// TODO close the holder (which will close the backends) if anything fails after this
 
 		TypeMetadataCollectorImpl metadataCollector = new TypeMetadataCollectorImpl();
-		contributors.forEach( c -> c.contribute( metadataCollector ) );
+		contributors.forEach( c -> c.contribute( buildContext, metadataCollector ) );
```

Comment:
I don't know the requirements of my future self. I just know that having a build broken at a certain point in time is pretty bad and can be annoying for my future self so I usually try to avoid that when I can.